### PR TITLE
Tutorial revision

### PR DESCRIPTION
**Pull request by JuvenalDuarte**
#### Please provide details about this Pull Request (why the change is being made, what this commit will do):
This PR brings all the files related to the documentation of the Machine Learning Tutorial on Carol, which is the AIKR02 for H2 2021.
The content is composed both by documentation (.md and .rst) files, together with sample code and configuration files to create functional example apps. None of the files have any impact on the package's core functionality.

### CODE REVIEW

File: tutorial/chapters/ch4_basic_batchapp/manifest.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          "en-US": "BHP Train Model NB"
+        },
+        "algorithmDescription": {
+          "pt-br": "BHP Train Model NB",
+          "en-US": "BHP Train Model NB"
+        },
+        "instanceProperties": {
+          "profile": "",
+          "properties": {
+            "dockerImage": "bhptrainmodelnb/bhptrainmodelnb:1.0.0",
```
Comment (rafarui) [inline tutorial/chapters/ch4_basic_batchapp/manifest.json:19]: "dockerImage" is deprecated no? it should be "dockerName", using dockerName we dont need to add the app name.. also no need for tag.. i.e.m 
`"dockerImage": "bhptrainmodelnb",`

File: tutorial/chapters/ch5_enhanced_batchapp/README.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+```
+
+Finally, we just need to adjust the `logging.cfg` (sample here) and `luigi.cfg` files, being the first for log standards and the second to define how the orchestration will behave. All the available configuration options on Luigi are listed on [this section](https://luigi.readthedocs.io/en/stable/configuration.html#:~:text=option%0Amysection().intoption-,Configurable%20options,-Luigi%20comes%20with) of the manual.
+
+The final directory structure should be:
+
+```bash
+/app/
+/app/flow/
+/app/flow/init.py
```
Comment (rafarui) [inline tutorial/chapters/ch5_enhanced_batchapp/README.md:235]: `__init__.py`

File: tutorial/chapters/ch5_enhanced_batchapp/manifest.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          "en-US": "BHP Train Model LG"
+        },
+        "algorithmDescription": {
+          "pt-br": "BHP Train Model LG",
+          "en-US": "BHP Train Model LG"
+        },
+        "instanceProperties": {
+          "profile": "",
+          "properties": {
+            "dockerImage": "bhptrainmodellg/bhptrainmodellg:1.0.0",
```
Comment (rafarui) [inline tutorial/chapters/ch5_enhanced_batchapp/manifest.json:19]: `dockerName`

**Review by rafarui — CHANGES_REQUESTED** [review]

File: doc/source/pycarol/tutorial/tutorial.enhancedbatchapp.rst [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+    /app/
+    /app/flow/
+    /app/flow/init.py
+    /app/flow/commons.py
+    /app/flow/ingestion.py
+    /app/flow/train_model.py
+    /app/flow/evaluate_model.py
+    /app/flow/deploy_model.py
+    /app/functions/
+    /app/functions/init.py
```
[the remark this answers is not in the document]
Comment (JuvenalDuarte) [reply doc/source/pycarol/tutorial/tutorial.enhancedbatchapp.rst]: Fixed on the docs.

File: doc/source/pycarol/tutorial/tutorial.enhancedbatchapp.rst [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+section <https://luigi.readthedocs.io/en/stable/configuration.html#:~:text=option%0Amysection().intoption-,Configurable%20options,-Luigi%20comes%20with>`__
+of the manual.
+
+The final directory structure should be:
+
+.. code:: bash
+
+    /app/
+    /app/flow/
+    /app/flow/init.py
```
[the remark this answers is not in the document]
Comment (JuvenalDuarte) [reply doc/source/pycarol/tutorial/tutorial.enhancedbatchapp.rst]: Fixed on the docs.

File: tutorial/chapters/ch4_basic_batchapp/manifest.json (hunk 1 above)
Comment (JuvenalDuarte) [reply tutorial/chapters/ch4_basic_batchapp/manifest.json]: Changed both on config and doc figures.

File: tutorial/chapters/ch5_enhanced_batchapp/README.md (hunk 2 above)
Comment (JuvenalDuarte) [reply tutorial/chapters/ch5_enhanced_batchapp/README.md]: Changed on docs.

File: tutorial/chapters/ch5_enhanced_batchapp/manifest.json (hunk 3 above)
Comment (JuvenalDuarte) [reply tutorial/chapters/ch5_enhanced_batchapp/manifest.json]: Changed both on config and doc figures.

**Pull request merged**